File: Linguagem C/Lista001_intro/Ex_002.c
//2) Fazer um algoritmo para calcular a área de um círculo. Sabe-se que a área do círculo é dada pelo produto do
//raio ao quadrado pela constante π = 3.14. Nos fluxograma e pseudocódigo, represente π como ‘pi’.
#include <stdio.h>
int main ()
{   
    float raio, area;
    scanf("%f",&raio);

    area = acos(-1)* raio*raio;
    printf("%0.2f", area);

return 0;

}

File: Python/001_Lista_intro/Ex_002.py
"""
2) Fazer um algoritmo para calcular a área de um círculo. Sabe-se que a área do círculo é dada pelo produto do 
raio ao quadrado pela constante π = 3.14. Nos fluxograma e pseudocódigo, represente π como 'pi'.
"""
print("Digite o raio do circulo: ")
raio = float(input())
pi = 3.14

area = raio**2*pi
print("A area do circulo sera igual a: {:.2f}".format(area))